libs/platform/ebpf_epoch.c

```diff
@@ -535,50 +486,118 @@ ebpf_epoch_cancel_work_item(_Inout_ ebpf_epoch_work_item_t* work_item)
     ExReleaseRundownProtection(&_ebpf_epoch_work_item_rundown_ref);
 }
 
+void
+ebpf_epoch_flush()
```

Comment:
Can you please add a description for this?

Reply:
It has doxygen in ebpf_epoch.h